### DISCUSSION TITLE
Cannot connect to RSC

### DISCUSSION THREAD

**Discussion by sayujshrestha** [rubrikinc/rubrik-powershell-sdk#25] [General]
I followed exactly as the readme says. Installed the module and ran Set-RscServiceAccountFile to generate an encrypted service account file and store it in a default location. But when I run Connect-Rsc command it gives me the following error. Am I missing anything. Note: I have MFA enabled in RSC.

PS C:\Windows\system32> Connect-Rsc
Connect-Rsc : One or more errors occurred.
At line:1 char:1
+ Connect-Rsc
+ ~~~~~~~~~~~
    + CategoryInfo          : AuthenticationError: (:) [Connect-Rsc], AggregateException
    + FullyQualifiedErrorId : UnableToAuthenticateToRubrikSecurityCloud,RubrikSecurityCloud.PowerShell.Cmdlets.Connect
   _Rsc

**Comment by guirava** [comment]
please try again with release 1.0 - please re-open if you are still experiencing issues.
